tests/Unit/ContactTest.php

```diff
@@ -586,20 +596,46 @@ public function testTotalOutstandingDebtAmountIsCorrect()
         /** @var Contact $contact */
         $contact = factory(Contact::class)->create();
 
-        $contact->debts()->save(new Debt(['in_debt' => 'no', 'amount' => 100]));
-        $contact->debts()->save(new Debt(['in_debt' => 'no', 'amount' => 100]));
+        $contact->debts()->save(new Debt([
+            'in_debt' => 'no',
+            'amount' => 100,
+            'account_id' => $contact->account_id,
+            'contact_id' => $contact->id,
+        ]));
+        $contact->debts()->save(new Debt([
+            'in_debt' => 'no',
+            'amount' => 100,
+            'account_id' => $contact->account_id,
+            'contact_id' => $contact->id,
+        ]));
 
         $this->assertEquals(200, $contact->totalOutstandingDebtAmount());
 
-        $contact->debts()->save(new Debt(['in_debt' => 'yes', 'amount' => 100]));
+        $contact->debts()->save(new Debt([
+            'in_debt' => 'yes',
+            'amount' => 100,
+            'account_id' => $contact->account_id,
+            'contact_id' => $contact->id,
```

Comment:
You don't need that. `$contact->debts...` will already set the contact_id for you

Reply:
Yes, but it will not set the account_id. I can remove the `contact_id` line if you want, but the `account_id` line still needs to be there.

Reply:
Awesome.